src/Model/Cms/AddCompetitionDTO.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace BalticRobo\Api\Model\Cms;
+
+
+use Symfony\Component\HttpFoundation\Request;
+
+class AddCompetitionDTO
+{
+    private $display_name;
+    private $name;
+    private $year;
+    private $registartionType;
+
+    private function __construct()
+    {
+    }
+
+    public static function createFromRequestAndYear(Request $request, int $year): self{
+        $content = json_decode($request->getContent(), true);

```

Comment:
Don't take JSON request. You have special event (https://github.com/balticrobo/website-api/blob/master/src/Event/JsonRequestTransformerSubscriber.php) which decodes JSON and put it in request bag.

```suggestion
        $content = $request->request->all();
```